packages/jest-runtime/src/script_transformer.js

```diff
@@ -261,7 +259,7 @@ export default class ScriptTransformer {
       code = transformed.code;
     }
 
-    if (instrument && mapCoverage && transformed.map) {
+    if (instrument && transformed.map) {
```

Comment:
For proper stack trace support, the instrument check should probably be killed as well. Test files are (almost) never instrumented, so line numbers in stacktraces for errors won't be fully correct otherwise.

Reply:
Agreed. If map is there save it, otherwise don't.

Reply:
Can you remove `instrument`?

Reply:
Done!